src/script/user/UserRepository.ts

```diff
@@ -587,24 +586,22 @@ export class UserRepository {
 
   /**
    * Check for users locally and fetch them from the server otherwise.
-   * @param {Array<string>} user_ids - User IDs
-   * @param {boolean} offline - Should we only look for cached contacts
-   * @returns {Promise<Array<User>>} Resolves with an array of users
+   * @param offline - Should we only look for cached contacts
    */
-  get_users_by_id(user_ids = [], offline = false) {
+  get_users_by_id(user_ids: string[] = [], offline: boolean = false): Promise<User[]> {
     if (!user_ids.length) {
       return Promise.resolve([]);
     }
 
-    const _find_user = user_id => {
+    const _find_user = (user_id: string) => {
       return this.findUserById(user_id) || user_id;
     };
 
     const find_users = user_ids.map(user_id => _find_user(user_id));
 
     return Promise.all(find_users).then(resolve_array => {
-      const known_user_ets = resolve_array.filter(array_item => !_.isString(array_item));
-      const unknown_user_ids = resolve_array.filter(array_item => _.isString(array_item));
+      const known_user_ets = resolve_array.filter(array_item => typeof array_item !== 'string') as User[];
```

Comment:
Please keep the underscore check, it does more than just `typeof`. 

In this case a `new String` will return `false`.

```
typeof "test"; // => "string"
typeof new String("test"); // => object
```